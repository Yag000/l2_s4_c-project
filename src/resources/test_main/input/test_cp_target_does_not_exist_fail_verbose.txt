mkdir t
cp t tt/k
