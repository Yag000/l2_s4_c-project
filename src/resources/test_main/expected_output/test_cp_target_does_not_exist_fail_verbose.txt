Parsing file src/resources/test_main/input/test_cp_target_does_not_exist_fail_verbose.txt ...
/$ mkdir t
/$ cp t tt/k
Invalid path to create a copy.
